Cargo.toml

```diff
@@ -89,6 +89,13 @@ rstest = "0.15"
 serde_yaml = "0.9"
 similar-asserts = "1"
 
+# Make it easy to view log output for select tests.
+# Set log level for tests with RUST_LOG=<level>, use with failed tests or
+# disable test stdout/stderr capture (`cargo test -- --nocapture` / `cargo
+# nextest run --no-capture`)
```

Comment:
Maybe we should start a new document or a wiki page for development practices.
Your tutorial on how to catch and debug flaky tests would also be good to have in there :)

Reply:
Sounds like a plan! Perhaps add a DEVELOPMENT.md pointing to such a page?

Reply:
Sounds good to me!

Reply:
I've made a start here: https://github.com/Nukesor/pueue/wiki/Development

Shall I leave it up to you to fill in a DEVELOPMENT.md file?

Reply:
I'll extend the wiki entry and transfer it to the DEVELOPMENT.md lateron :)